config/ingest.go

```diff
@@ -42,6 +53,7 @@ func NewIngest() Ingest {
 		PubSubTopic:             "/indexer/ingest/mainnet",
 		StoreBatchSize:          4096,
 		SyncTimeout:             Duration(2 * time.Hour),
+		SyncSegmentDepthLimit:   1000,
```

Comment:
The value is entirely made up; setting it would need some measurements. Please shout if you have a more reasonable initial value in mind 🙇

Reply:
Our EC2 dev node was able to get to around 15000 advertisements before it died.  So, I think 10000 for prod is reasonable.